tool_commands/raster_checker/raster_checker_log.py

```diff
@@ -213,6 +214,9 @@ def result_per_check_to_msg(self, result_row):
         if feedback_level == "error":
             self.nr_error_logrows += 1
 
+        if feedback_level == "warning":
+            self.nr_warning_logrows += 1
+
```

Comment:
Wordt dit ergens gebruikt?

Reply:
in pop_up_finished_or_question() 
nr_errors en nr_warnings wordt nu geprint in pop-up (als checker klaar is)